src/w3d/lib/cpudetect.cpp

```diff
@@ -273,6 +273,11 @@ void Get_OS_Info(OSInfoStruct &os_info,
             os_info.VersionString = "UNKNOWN";
             break;
     }
+#else
+    memset(&os_info, 0, sizeof(os_info));
```

Comment:
This is c++ can't you just do `os_info = OSInfoStruct {};`

Reply:
I copied this from 5 lines above

Reply:
Oh yes so you did. just leave as is.

Reply:
I still fixed this in my last commit